src/liballoc/raw_vec.rs

```diff
@@ -318,12 +318,28 @@ impl<T, A: Alloc> RawVec<T, A> {
                     }
                 }
                 None => {
-                    // skip to 4 because tiny Vec's are dumb; but not if that
-                    // would cause overflow
-                    let new_cap = if elem_size > (!0) / 8 { 1 } else { 4 };
-                    match self.a.alloc_array::<T>(new_cap) {
-                        Ok(ptr) => (new_cap, ptr),
-                        Err(e) => self.a.oom(e),
+                    if used_cap == 0 {
+                        // skip to 4 because tiny Vec's are dumb; but not if that
```

Comment:
Isn't it "Vecs", not "Vec's"?

You only need the apostrophe if the "Vec" is possessing something.

Reply:
this is a pre-existing comment, but yes